### PR TITLE
Fixing preview button text in comment section

### PR DESCRIPTION
**Pull request by Rafi993**
## What type of PR is this?

- [ ] Refactor
- [ ] Feature
- [x] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description

Preview button text in comment section did not reset back after the comment is submitted this PR fixes it

## Related Tickets & Documents

closes https://github.com/forem/forem/issues/13182

## QA Instructions, Screenshots, Recordings

Comment

### UI accessibility concerns?

_If your PR includes UI changes, please replace this line with details on how
accessibility is impacted and tested. For more info, check out the
[Forem Accessibility Docs](https://docs.forem.com/frontend/accessibility)._

## Added tests?

- [ ] Yes
- [x] No, and this is why: _please replace this line with details on why tests
      have not been included_
- [ ] I need help with writing tests

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

- [ ] I've updated the [Developer Docs](https://docs.forem.com) and/or
      [Admin Guide](https://forem.gitbook.io/forem-admin-guide/), or
      [Storybook](https://storybook.forem.com/) (for Crayons components)
- [ ] I've updated the README or added inline documentation
- [ ] I will share this change in a [Changelog](https://forem.dev/t/changelog)
      or in a [forem.dev](http://forem.dev) post
- [ ] I will share this change internally with the appropriate teams
- [ ] I'm not sure how best to communicate this change and need help
- [ ] This change does not need to be communicated, and this is why not: _please
      replace this line with details on why this change doesn't need to be
      shared_

## What gif best describes this PR or how it makes you feel?

Happy

### CODE REVIEW

File: app/assets/javascripts/initializers/initializeCommentsPage.js.erb [hunk 1]
```diff
@@ -268,8 +268,9 @@ function handleCommentSubmit(event) {
             });
 
             var preview = document.getElementById("preview-div");
-            preview.classList.add("preview-toggle");
+            var togglePreview = document.querySelector('.preview-toggle');
```
Comment (Link2Twenty) [inline app/assets/javascripts/initializers/initializeCommentsPage.js.erb:271]: Where possible it would be good to use `const` rather than `var`.

**Comment by Link2Twenty** [comment]
Looks like a flaky test, causing an unrelated fail.

File: app/assets/javascripts/initializers/initializeCommentsPage.js.erb (hunk 1 above)
Comment (Rafi993) [reply app/assets/javascripts/initializers/initializeCommentsPage.js.erb:271]: Does `js.erb` get transpiled to older js versions ?

Comment (rhymes) [reply app/assets/javascripts/initializers/initializeCommentsPage.js.erb:271]: @Rafi993 we don't support IE11 and `const` doesn't need transpilation: https://caniuse.com/const

Comment (Rafi993) [reply app/assets/javascripts/initializers/initializeCommentsPage.js.erb]: I have now replaced `var` with `const`. I think there are few vars in the codebase that is used in few files they could also be replaced with `const` in a separate PR. I'll create issue for that

**Review by rhymes — APPROVED** [review]
Works well! Thanks @Rafi993!

**Review by Zhao-Andy — APPROVED** [review]
Works like a charm! Thanks for the PR :)

**Pull request merged**